### PR TITLE
Add gitignore file

### PR DESCRIPTION
**Pull request by tashisrael**
Request to view this "Hello Microverse" project and give advice were neccessary

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (levy002) [inline test.md]: - [ ] Kindly, make sure to add this `test.md` file into `.gitignore` in order to not be pushed on your GitHub repository as required in this project requirements. :wink:

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#ignore ALL .md files
+*.md
```
Comment (levy002) [inline .gitignore]: - [ ] Kindly, make sure to add the `test.md` file into `.gitignore` in order to not be pushed on your GitHub repository as required in this project requirements. :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @tashisrael, 
### I'm Levy, your PR reviewer

Good job so far! :boom:
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Descriptive README :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- Kindly, make sure that your linters are passing (all should look green)
- Kindly, make sure to add a pull request description
- Kindly make sure to write your commit message in imperative tense and capitalized. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tashisrael — COMMENTED** [review]
Request to review this "Hello Microverse" project. All corrections have been done

File: .gitignore (hunk 2 above)
Comment (tashisrael) [reply .gitignore]: Thank you for the feedback. Am done making the changes and ready for review

**Comment by tashisrael** [comment]
> ## Hi @tashisrael,
> ### I'm Levy, your PR reviewer
> Good job so far! 💥 There are some issues that you still need to work on to go to the next project but you are almost there!  [ 68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966 ](https://camo.githubusercontent.com/77566116408d88259ddadf7605a02b67a4e825e002a39682ca5f2e8bca8c4d64/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) [ ](https://camo.githubusercontent.com/77566116408d88259ddadf7605a02b67a4e825e002a39682ca5f2e8bca8c4d64/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)
> 
> ## Highlights
> * Descriptive README ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> * Kindly, make sure that your linters are passing (all should look green)
> * Kindly, make sure to add a pull request description
> * Kindly make sure to write your commit message in imperative tense and capitalized. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @levy002 in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._


Thank you for the feedback. Am done making the changes and ready for review @levy002

**Review by prolajumokeoni — APPROVED** [review]
Hi @tashisrael 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**